setup.py

```diff
@@ -71,7 +71,8 @@ def run(self):
                             'stuart_pr_eval=edk2toolext.invocables.edk2_pr_eval:main',
                             'omnicache=edk2toolext.omnicache:main',
                             'nuget-publish=edk2toolext.nuget_publishing:go',
-                            'edk2_capsule_tool=edk2toolext.capsule.capsule_tool:main']
+                            'edk2_capsule_tool=edk2toolext.capsule.capsule_tool:main',
+                            'versioninfo_tool=edk2toolext.versioninfo.versioninfo_tool:main']
     },
     install_requires=[
```

Comment:
don't we need pefile listed here?  The pip requirements listed in txt file are for development and publishing.  For actual runtime we need to list dependencies here.